ztp/source-crs/ClusterLogSubscription.yaml

```diff
@@ -3,6 +3,8 @@ kind: Subscription
 metadata:
   name: cluster-logging
   namespace: openshift-logging
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "1"
```

Comment:
Need to confirm that a subscription will eventually get reconciled if the ns/operatorgroup has not. This use to be a problem need to confirm it has been resolved